build.sbt

```diff
@@ -34,7 +42,7 @@ val installAll =
      |&& gem install activesupport
      |&& gem install parser:2.4.0.0
      |&& gem install pry
-     |&& gem install rubocop:0.49.0
+     |&& gem install rubocop:${rubocopVersion}
```

Comment:
Hm, since the idea was having the version in a unique place I left it implied that it would have to exist. But I'll add a default version for safety